soc/arm/nxp_imx/rt5xx/soc.c

```diff
@@ -288,6 +288,16 @@ static void clock_init(void)
 #if CONFIG_USB_DC_NXP_LPCIP3511
 	usb_device_clock_init();
 #endif
+
+#if (DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(flexcomm1), nxp_lpc_i2s, okay))
```

Comment:
Please add `CONFIG_I2S` to the check here- this way, the clocks are not setup unless the user also enables I2S functionality